rocAL/rocAL/source/decoders/image/turbo_jpeg_decoder.cpp

```diff
@@ -112,7 +112,7 @@ Decoder::Status TJDecoder::decode(unsigned char *input_buffer, size_t input_size
                             max_decoded_width * planes,
                             max_decoded_height,
                             tjpf,
-                            TJFLAG_FASTDCT,
+                            TJFLAG_ACCURATEDCT,
```

Comment:
Would suggest having this a parameter from user like DALI to decoded using
FASTDCT / ACCURATEDCT,

Reply:
Discussed on the changes needed for adding this option - planned to issue a separate PR if a need arises for usage of FastDCT algorithm